logger.ts

```diff
@@ -4,6 +4,6 @@ import config from './server/config'
 
 const formatOut = bunyanFormat({ outputMode: 'short', color: !config.production })
 
-const logger = bunyan.createLogger({ name: 'Hmpps Contacts Ui', stream: formatOut, level: 'debug' })
+const logger = bunyan.createLogger({ name: 'Digital Prison Services', stream: formatOut, level: 'debug' })
```

Comment:
I think this should be still 'hmpps contacts UI' ?

Reply:
I think this should still stay as it was before because it is the name for the logger for this module, we will see this see logs on the console with this name hence easily stand out from the other apps